cli-feed-v3.json

```diff
@@ -443,6 +443,45 @@
         }
       ]
     },
+    "2.11.1": {
+      "Microsoft.NET.Sdk.Functions": "1.0.23",
+      "cli": "https://functionscdn.azureedge.net/public/2.2.27/Azure.Functions.Cli.win-x86.2.2.27.zip",
+      "sha2": "96855F053896DBABC042729982DDDA36A3B5AE261862F1A9FF8A181FB1F09806",
+      "nodeVersion": "8.11.1",
+      "localEntryPoint": "func.exe",
+      "itemTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-10300",

```

Comment:
We have some template changes..i will provide new template pacakge

Reply:
https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-10310